webpack.config.js

```diff
@@ -30,20 +32,15 @@ module.exports = {
           plugins: ['transform-runtime']
         }
       },
-      { test: /\.css$/,
-        include: path.resolve(__dirname, 'static_src/css'),
+      {
+        test: /\.css$/,
+        include: [
+          path.resolve(__dirname, 'static_src/css'),
+          path.resolve(__dirname, 'node_modules/cloudgov-style'),
+          CG_STYLE_PATH
```

Comment:
Gotcha. No, I don't think so, because the proper files will be resolved in `node_modules/cloudgov-style`. Just so that its not undefined though I should probably put in a default string or something.